CMakeLists.txt

```diff
@@ -847,21 +847,17 @@ else()
 	include(DockerConfig)
 endif()
 
-if(NOT WIN32)
 # Create a custom build target that will run the linter.
 get_property(extensions GLOBAL PROPERTY EXTENSION-LINTERS)
 add_custom_target(linter
-	COMMAND ${CMAKE_SOURCE_DIR}/thirdparty/google-styleguide/run_linter.sh
-			${CMAKE_SOURCE_DIR}/libminifi/include/ --
-			${CMAKE_SOURCE_DIR}/libminifi/src/
-	COMMAND ${CMAKE_SOURCE_DIR}/thirdparty/google-styleguide/run_linter.sh
-			${CMAKE_SOURCE_DIR}/libminifi/include/ --
-			${CMAKE_SOURCE_DIR}/libminifi/test/
-	COMMAND ${CMAKE_SOURCE_DIR}/thirdparty/google-styleguide/run_linter.sh
-			${CMAKE_SOURCE_DIR}/encrypt-config/ --
-			${CMAKE_SOURCE_DIR}/encrypt-config/
+	COMMAND python ${CMAKE_SOURCE_DIR}/thirdparty/google-styleguide/run_linter.py -i
```

Comment:
Since the run_linter.py only checked the cpp files in the first argument and only the h files in the second, we didnt check any h files in the libminifi/test folder.
It also needlesly double checked the libminifi/include folder.

The cpplint.py doesnt require the filelist to be separated by header/source, so its easier to just have one argument.